cmd/zc_attr_filter_windows.go

```diff
@@ -0,0 +1,105 @@
+// Copyright © 2017 Microsoft <wastore@microsoft.com>
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package cmd
+
+import (
+	"path/filepath"
+	"syscall"
+	"strings"
+)
+
+type attrFilter struct {
+	fileAttributes uint32
+	filePath       string
+	resultIfMatch  bool
```

Comment:
For clarity's sake, I'd recommend calling this "include". I felt initially a bit confused by the actual behavior of this flag until I did a little reading.

Reply:
That being said, there's also totally an argument to be had that `resultIfMatch` makes more sense-- as this is more literally what it does. My view is just that `resultIfMatch` is just a bit of an obtuse way to say "include these attributes"

---
In reply to: [305002730](https://github.com/Azure/azure-storage-azcopy/pull/501#discussion_r305002730) [](ancestors = 305002730)

Reply:
What about `isIncludeFilter` - since the word `include` on its own may also initially confuse some folks.  FWIW, I like `isIncludeFilter` better than `resultIfMatch`

Reply:
Yeah. I like `isIncludeFilter` better, too. I don't care too much about what exactly this Boolean should be called, as long as the name clarifies what the variable is meant to do.